plumbing/protocol/packp/updreq_decode.go

```diff
@@ -168,6 +162,9 @@ func (d *updReqDecoder) decodeCommands() error {
 
 func (d *updReqDecoder) decodeCommandAndCapabilities() error {
 	b := d.s.Bytes()
+	if bytes.Equal(b, pktline.Flush) {
+		return ErrFlushPacketRecieved
+	}
```

Comment:
This would also represent an empty `update-request` message, and due to that I am thinking we are better off reusing the existing `ErrEmpty`.